### ISSUE TITLE
Remove timestamp from "plugin.properties" + use linux-style file separator

### ISSUE THREAD

**Issue by raducoravu** [dita-ot/dita-ot#2193]
Every time after I run the DITA OT integrator, my SVN client detects and marks as outgoing in the repository the "plugin.properties" and the "dost-configuration.jar".
All this because the "plugin.properties" contains a timestamp which constantly gets modified.
Maybe we could try to remove that timestamp from the properties file when the integrator is run.
Also when running on Windows the plugin.properties contains references to resources using the "\" Windows style separator. So this would mean that if I share my DITA OT repository with somebody on Linux, they will have problems using the information in the plugin.properties.
The linux style path separator should be used on all platforms because the Windows Java VM supports both styles of path separators when creating File objects.

**Comment by jelovirt** [comment]
`Properties.store` method adds the timestamp, I don't know how to disable that.

Using *NIX path separators would mean we would have to normalize all values on read.

:+1: for both.

**Comment by raducoravu** [comment]
About the properties, indeed we would need either to pass it an output stream implementation which skips that part or maybe better have our own properties implementation which also sorts keys before they are flushed to disk (or maybe better, save them to XML as ANT can also load properties from XML).

**Comment by jelovirt** [comment]
You can read properties format with Ant, we don't need to use XML.

**Comment by raducoravu** [comment]
I know, we can probably continue to use them, at least the code will remain backward compatible. 
I'm not a big fan of properties files because they have a very limited encoding (ISO 8859-1) and all other characters are escaped, making them hard to be human readable, for example here's how a Japanese value looks in a properties files:

      Add_first_choice_particle=\u6700\u521D\u306E\u9078\u629E\u80A2\u306E\u8FFD\u52A0

**Comment by jelovirt** [comment]
I would prefer to not use Java properties, but since that's what Ant supports. We could add support for some configuration file format, but then we'd need to add custom Ant tasks to read them into Ant properties. In Java code it would not be that bad. We could use e.g. https://github.com/typesafehub/config that's a superset of JSON, thus pretty familiar format; or then just stick to JSON. I would prefer not use TOML/YAML/INI.

**Comment by raducoravu** [comment]
Sure, no problem if we continue using the properties format.

**Comment by jelovirt** [comment]
I think my current preference would be https://github.com/typesafehub/config#using-hocon-the-json-superset and keep it as close to either JSON or Properties format as possible. And, use *NIX directory separators.

**Comment by xephon2** [comment]
> but then we'd need to add custom Ant tasks to read them into Ant properties
@jelovirt : Will you provide an ANT-based getter and setter solution? I think that it is important, that someone without real programming skills (Java, etc.) can read/write from/to a properties file.

**Comment by jelovirt** [comment]
@xephon2 Yes, we'd add a task to read those configuration files in and have them be available for Ant code to use. Since this is the main configuration file, we would not add a writer task, because those files don't need to be written at run-time.

**Comment by xephon2** [comment]
:thumbsup:

**Comment by easirois** [comment]
Hi, We have run into this issue ourselves for our hosted environment and one of our clients has hit it as well while using the dita command to run the integrator on Windows and failing to run properly on the Linux side of the house.  The issue is the plugin.properties having Windows-style separator.  I can open up a new ticket instead of reopening this one if folks prefer.

@raducoravu @jelovirt @robander

**Comment by raducoravu** [comment]
I'm reopening the issue, I think it's easier in order to preserve all comments.
